modules/flight_input/flight_interface.py

```diff
@@ -0,0 +1,45 @@
+"""
+Creates flight controller and combines odometry data and timestamp.
+"""
+
+from modules.common.mavlink.modules import flight_controller
+from modules import odometry_and_time
+
+
+# This is just an interface
+# pylint: disable=too-few-public-methods
+class FlightInterface:
+    """
+    Create flight controller and combines odometry data and timestamp.
+    """
+    __create_key = object()
+
+    @classmethod
+    def create(cls, address: str) -> "tuple[bool, FlightInterface | None]":
+        """
+        address, TCP or UDP string.
+        """ 
+        if address == "":
+            return False, None
+        
+        assert cls.__create_key is FlightInterface.__create_key, "Use create() method"
```

Comment:
Move this back to the constructor, and move the flight controller creation to here. If the flight controller failed, return `False, None` .